### PR TITLE
Fix issues for multiple projects

### PR DESCRIPTION
**Pull request by henryrneh**
Previous Jazzer version only instrument code under the same package as the fuzz test, this PR adds `--instrumentation_includes` to several projects.

Also fix build failures in jdom and jsqlparser, fix fuzzing blocker in qdox and avro, delete not needed file in apache-cxf

### CODE REVIEW

File: projects/jdom/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-jvm
 
-RUN curl -L https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.12-bin.tar.gz -o ant.tar.gz && \
```
Comment (DavidKorczynski) [inline projects/jdom/Dockerfile:19]: Just checking: some of the changes in this PR are changes not focused on the `--instrumentation_includes` as described in the PR text, is this intentional?

Comment (henryrneh) [reply projects/jdom/Dockerfile:19]: @DavidKorczynski Sorry I forgot to add that part into the description, it is updated now. Thanks for reminding :)

Comment (Alan32Liu) [reply projects/jdom/Dockerfile:19]: Thanks!
I will convert this to a draft for now, please feel free to ping me when it is ready : )

Comment (henryrneh) [reply projects/jdom/Dockerfile:19]: @Alan32Liu Thank you but actually this PR is ready to be merged, thanks for merging :)

**Review by DavidKorczynski — APPROVED** [review]

**Comment by henryrneh** [comment]
Thanks @DavidKorczynski @Alan32Liu !

**Pull request merged**